src/modules/Api.ts

```diff
@@ -75,6 +75,84 @@ export class Api {
     });
   }
 
+  /**
+   * Creates, but does not place a signed order and signed cancel pair
+   */
+  public async replaceOrder({
```

Comment:
Can we break this out into two functions: this one and one that takes an already signed order and cancel and just makes the API request?